Documentation/troubleshooting.rst

```diff
@@ -249,6 +250,72 @@ following output in ``cilium status``:
     See how to :ref:`ensure a pod is managed by Cilium<ensure_managed_pod>`
     for more details.
 
+Observing flows with Hubble Relay
```

Comment:
We seem use several different capitalizations of "Hubble Relay" across the docs:

```
$ git grep -i "hubble.relay"
contributing/development/codeoverview.rst:42:hubble-relay
contributing/development/codeoverview.rst:43:  Hubble relay server
contributing/development/codeoverview.rst:100:hubble-relay
contributing/development/codeoverview.rst:101:  Hubble relay agent
contributing/development/codeoverview.rst:125:pkg/hubble/relay
contributing/release/stable.rst:180:   ``cilium-docker-plugin`` and ``hubble-relay`` on DockerHub.
contributing/testing/e2e.rst:198:      -cilium.hubble-relay-image string
contributing/testing/e2e.rst:199:            Specifies which image of hubble-relay to use during tests
hubble.rst:10:          "Hubble embedded") and Hubble-relay components but does not cover the
hubble.rst:21:Hubble-relay. Hubble-relay is a standalone component which is aware of all
hubble.rst:24:multi-node. Hubble-relay's main goal is to offer a rich API that can be safely
hubble.rst:89:primarily used by Hubble-relay in order to have a cluster-wide view of all
hubble.rst:96:Hubble-relay
hubble.rst:99:.. note:: At the time of this writing, the hubble-relay component is still
hubble.rst:101:          internal documentation about Hubble-relay is limited.
hubble.rst:103:Hubble-relay is a component that was introduced in the context of multi-node
install/system_requirements.rst:300:4245/tcp                 hubble relay
```

Might make sense to unify these (at least when not talking about the `hubble-relay` binary). Not something that needs to be done as part of this PR, just something I noticed.

Reply:
Please, see #11923.